cypress/e2e/hub/execution-environments.cy.ts

```diff
@@ -204,4 +204,58 @@ describe('Execution Environment Details tab', () => {
     });
     cy.get('[data-cy="readme"]').contains('this should not be saved.').should('not.exist');
   });
+
+  it('should successfully sync execution environment from Docker registry', () => {
+    cy.createHubRemoteRegistry().then((remoteRegistry) => {
+      cy.createHubExecutionEnvironment({
+        executionEnvironment: {
+          include_tags: ['latest'],
+          registry: remoteRegistry.id,
+        },
+      }).then((executionEnvironment) => {
+        cy.syncRemoteExecutionEnvironment(executionEnvironment);
```

Comment:
@jerabekjiri we need to delete the created resources as well.  For all those tests.

```suggestion
        cy.syncRemoteExecutionEnvironment(executionEnvironment);
         cy.deleteHubExecutionEnvironment(executionEnvironment).then(() => {
         cy.deleteHubRemoteRegistry(remoteRegistry);
```

Reply:
Good point, forgot about that :+1:  Thanks